03-task-manager/starter/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.11.10"
```

Comment:
We ideally want a newer version of Mongoose. You probably see that you get an deprecation warning that includes the following text currently:
`Future versions of Node.js will throw an error.`

You could theoretically avoid future errors by not updating Node or Mongoose, but you're bound to run into issues that way, especially with the security of your API and database.

One way to update is to run the below from your `starter` directory, assuming you're using `npm` as your package manager:
`npm install mongoose@latest`